worksheet_C/README.md

```diff
@@ -0,0 +1,24 @@
+>>> Worksheet C: Computational Complexity
+
+A)	The algorithm performs a check for duplications in a list, with n being the length of the list. If a duplication is found then the procedure returns True, if no duplications are found in list then the procedure returns False.
+
+B)	The worst case running time is quadratic O(N)² because the program has a for loop nested within the first for loop and therefore, the amount of iterations needed increases by the square of the input size. The program would have been linear had the second list not been nested in the first, and adding a new loop would increase the number of iterations proportionally. However, with this quadratic algorithm, if n = 6, for every iteration of the first loop, the inner loop will perform 6 loops: with each increase of the input, the loops on the second iteration will be n².
+
+C)	The algorithm is still correct; the procedure now only runs through j as far as i – 1. The procedure will only check a value against values that came before it in the list. This stops the procedure from checking a value against the same value multiple times and stops a value being checked against itself.
+
+D)	The algorithm will run approximately twice as fast because the procedure no longer needs to perform as many iterations. The procedure no longer checks the same two values against each other more than once and no longer checks a value against itself. Overall this roughly halves the number of iterations.
```

Comment:
✅ (note you can be more specific than "more than once" -- the old procedure checks each pair exactly twice)